plugins/node/opentelemetry-instrumentation-fastify/src/instrumentation.ts

```diff
@@ -62,7 +62,7 @@ export class FastifyInstrumentation extends InstrumentationBase {
     return [
       new InstrumentationNodeModuleDefinition<any>(
         'fastify',
-        ['^3.0.0'],
+        ['>=3.0.0'],
```

Comment:
Support 3.x and 4.x.

Reply:
could you add a check to not try to patch v5 when it will comes out ?